txbits/app/wallet/Wallet.scala

```diff
@@ -304,6 +358,7 @@ class Wallet(rpc: JsonRpcHttpClient, currency: CryptoCurrency, nodeId: Int, para
   }
 
   private def getBalance = {
+    // This method is unused as balance is now tracked in the database
```

Comment:
Bad things. Every deposit and withdrawal is tracked in the database so the balance should be accurate.